tests/test_language.py

```diff
@@ -23,7 +24,7 @@ def test_init_invalid(self):
 
     def test_properties(self):
         lang = Language(self.python)
-        self.assertEqual(lang.version, 14)
+        self.assertEqual(lang.abi_version, 14)
```

Comment:
>_possible bug_

**✨ AI Suggestion:** The `cast` here seems unnecessary as `Language(self.python)` already returns a `Language` object. Consider removing the cast for better readability.
```suggestion
        lang = Language(self.python)
        self.assertEqual(lang.abi_version, 15)
```